### DISCUSSION TITLE
Connected to Guacamole. Waiting for response...

### DISCUSSION THREAD

**Discussion by Iven2132** [MaxWaldorf/guacamole#138] [Q&A]
Hi, I'm trying to run Guacamole on Modal (container environment) and I can access the Guacamole dashboard but when I create a new connection and try to access it I first get "Connected to Guacamole. Waiting for response..." for a bit then it gets disconnected. 

Can anyone please help me set things up? 

I created a VNC connection with the network with hostname: localhost and port 8080, I also tried 4822 and 5900 but it didn't work. I'm using "flcontainers/guacamole" Docker image.

**Comment by Iven2132** [comment]
Hi @MaxWaldorf Can you please help me?

**Reply by MaxWaldorf** [reply]
Sorry,

What you describe seems to be network related...

Nothing to do with my configuration of the container...
